tests/test_topsites.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""Tests for topsites script."""
+
+import datetime
+import os
+import sys
+import unittest
+from xml.dom.minidom import parseString
+
+from mock import patch
+
+# Add tools module to import path
+sys.path.append(os.path.realpath(os.pardir))
```

Comment:
the comment and the `sys.path…` are not necessary here.
And it means that you can get rid of `import os` and `import sys`